src/arc.ts

```diff
@@ -1,3 +1,5 @@
+/*global window: any */
```

Comment:
please remove that

Reply:
ok, removed it. You need to approve again...